psst-gui/src/ui/mod.rs

```diff
@@ -150,18 +150,18 @@ fn root_widget() -> impl Widget<AppState> {
         .with_flex_child(Overlay::bottom(route_widget(), alert_widget()), 1.0)
         .with_child(playback::panel_widget())
         .background(theme::BACKGROUND_LIGHT);
-
-    let split = Split::columns(sidebar, main)
-        .split_point(0.2)
-        .bar_size(1.0)
-        .min_size(150.0, 300.0)
-        .min_bar_area(1.0)
-        .solid_bar(true);
+        
+    let split = Flex::row()
+        .with_child(sidebar)
+        .with_flex_child(main, 1.0)
+        .with_child(queued::queue_widget())
+        .background(theme::BACKGROUND_DARK);
```

Comment:
Ah, so the side bar (with playlists) now doesn't follow a split ratio. Was there a reason that this logic was implemented as a fix or more so a preference?

Reply:
It was to follow consistency with the right-hand side as that currently doesn't move, and additionally to make it so the sidebar doesn't become unreasonably wide.

Reply:
I have now changed it to follow the split ratio!